gatsby-node.js

```diff
@@ -124,10 +106,42 @@ const createAuthors = async (createPage, slug) => {
   })
 }
 
+const createSections = async (createPage, section, slug) => {
```

Comment:
combine `createSections` and `createSection` together, put the code in `createSections` inside `createSection`. There should be only 1 function called `createSection`, otherwise, it's kinda confusing